src/DependencyInjection/CompilerPass/StimulusHelperPass.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Mercure Component project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Bundle\MercureBundle\DependencyInjection\CompilerPass;
+
+use Symfony\Component\DependencyInjection\Alias;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * Registers a dynamic alias to use a service from WebpackEncoreBundle or StimulusBundle.
+ */
+final class StimulusHelperPass implements CompilerPassInterface
+{
+    public function process(ContainerBuilder $container)
+    {
+        if ($container->hasDefinition('stimulus.helper')) {
+            $id = 'stimulus.helper';
+        } else {
+            $id = 'webpack_encore.twig_stimulus_extension';
+        }
+        $container->setAlias('turbo.mercure.stimulus_helper', new Alias($id));
```

Comment:
It seems possible that none of WebpackEncoreBundle and StimulusBundle are available, isn't it? If yes then this compiler pass would need to be registered conditionally - as done in MercureExtension for the TurboStreamListenRenderer service - avoiding a compile error

Reply:
Thanks - good call!

Reply:
Fixed up now. And it made me realize that the `conflict` was not needed. IF `symfony/ux-turbo` is installed (in ANY version), then we are guaranteed that at least one of `symfony/webpack-encore-bundle` or `symfony/stimulus-bundle` is installed. So I've removed the conflict - but added the same check here as in `MercureExtension`.